packages/icons/icons.json

```
@@ -933,217 +933,217 @@
     {
         "displayName": "Helper management",
         "iconName": "helper-management",
-        "tags": "square, widget",
+        "tags": "square, widget, graph, chart, data, document, blocks, spreadsheet, format, right bottom justify, loading, snake",
         "group": "interface",
         "codepoint": 61933
     },
     {
         "displayName": "Send to",
         "iconName": "send-to",
-        "tags": "circle, export, arrow",
+        "tags": "circle, export, arrow, outside, external, pop out, open object, open item, ellipses, array",
         "group": "action",
         "codepoint": 62121
     },
     {
         "displayName": "Eye",
         "iconName": "eye-open",
-        "tags": "show, visible, clear, view, vision",
+        "tags": "show, visible, clear, view, vision, eye, see, eye and check",
```

```suggestion
        "tags": "show, visible, clear, view, vision, see",
```

there's no "check" here. and "eye" is redundant

---

I think the idea behind "check" is that this icon can be used to denote a review or check action (something needs eyes on it)